api/errors/builder.go

```diff
@@ -0,0 +1,93 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/builder component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeAtLeastOneArg is generated when the parser comes across a
+	// Dockerfile command that doesn't have any args.
+	ErrorCodeAtLeastOneArg = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
I'm not following, how is this is mis-use?  All I did was convert existing errors into ErrCodes. While we may be able to tweak the error messages, I don't think that should be the point of this initial exercise. I'd prefer to have zero UX impact on the first pass of this, meaning just get errcode in there.  Then we can revisit, and potentially consolidate, error messages, but that's just a clean-up/optimization to me.  Or are you referring to something else?